### ISSUE TITLE
Build issue while installing with yarn 2

### ISSUE THREAD

**Issue by joshua1** [kaisermann/svelte-preprocess#110]
**Describe the bug**
I Tried installing packages on a svelte project using the newly released yarn 2 and got this error on svelte=preprocess install 
``` YN0009: │ svelte-preprocess@npm:3.3.0 [d7ed1] couldn't be built successfully (exit code 1, logs can be found here: /var/folders/lr/_h2v_4v54rj9zy3ymz_f386c0000gn/T/logfile-46571udcgo1h31zer.log)
```

**Logs**
```
# This file contains the result of Yarn building a package (svelte-preprocess@virtual:d7ed1848a21c820ecedeeb8d612b3590fd625b1c6481824b087a78535b107fd21109966aa2635f5580f2b9f4905e7ec8dab5233b4e9a0b835de69dcbaffc7063#npm:3.3.0)
# Script name: postinstall

[eval]:1
console.log('u001b[36mu001b[1m[svelte-preprocess] Don't forget to install the preprocessors packages that will be used: u001b[22mu001b[39mu001b[34mnode-sass/sass, stylus, less, postcss &  postcss-load-config, coffeescript, pug, etc...u001b[0m')
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SyntaxError: missing ) after argument list
    at new Script (vm.js:88:7)
    at createScript (vm.js:262:10)
    at Object.runInThisContext (vm.js:310:10)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1139:30)
    at evalScript (internal/process/execution.js:94:25)
    at internal/main/eval_string.js:23:3
```
**To Reproduce**
install yarn 2
yarn add svelte-preprocess in your sveltte project

**Expected behavior**

**Information about your project:**

- Your browser and the version: Brave 1.2.43

- Your operating system: OSX 10.15.2

- `svelte-preprocess` version: 3.3.0

- Whether your project uses Webpack or Rollup: Rollup

**Comment by kaisermann** [comment]
Thanks for the warning! Fixed in `v3.3.1` 🎉

**Comment by andrewdc** [comment]
Tested with 3.3.1. Fixed. Many thanks @kaisermann!!
